### PR TITLE
[Security] Latest SAST Scanning Results on v18.0.0 - highs

### PR DESCRIPTION
**Pull request by CobyPear**
Upgrade packages with high vulnerabilities found through `npm audit --production` and Veracode scan.

## Description / Motivation

upgraded some dependencies in packages and samples

maintain best practices for a securing our NPM packages and sample apps against potential exploits.

## Testing Details

built samples and packages, ran tests.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)
- [x] Security, non-breaking

### CODE REVIEW

File: packages/sitecore-jss-nextjs/package-lock.json [hunk 1]
```diff
@@ -1,10715 +1,6696 @@
 {
```
Comment (ambrauer) [inline packages/sitecore-jss-nextjs/package-lock.json:1]: Looks like something's not quite right with `sitecore-jss-nextjs`. I'm seeing:

- A merge conflict on the lock file
- Errors on ` npm run build`
- The lock file indentation formatting changed, which makes it difficult to see what actually changed

**Review by ambrauer — CHANGES_REQUESTED** [review]
See comment re: `sitecore-jss-nextjs`. Otherwise, this is looking good.

File: packages/sitecore-jss-nextjs/package-lock.json (hunk 1 above)
Comment (CobyPear) [reply packages/sitecore-jss-nextjs/package-lock.json:1]: These should be resolved now.

**Review by ambrauer — APPROVED** [review]
Looks good!

**Pull request merged**